### PR TITLE
Added worfklow for cloning PRs

### PR DESCRIPTION
**Pull request by bluvulture**
## Changes in this pull request  
Added workflow for cloning PRs to ee repository using reusable PR clone action 
https://github.com/pimcore/workflows-collection-public/blob/v1.0.0-rc.1/.github/workflows/reusable-pr-clone.yml

### <samp>🤖 Generated by Copilot at 3892ae5</samp>

Add a new workflow to clone closed pull requests with a `clone` label. The workflow uses the `.github/workflows/clone-pr.yml` file and checks the labels and the branch name of the closed pull request.

### <samp>🤖 Generated by Copilot at 3892ae5</samp>

> _`clone-pr.yml` file_
> _When pull request is closed_
> _Autumn leaves fall down_

### <samp>🤖 Generated by Copilot at 3892ae5</samp>

* Add a new workflow to clone closed pull requests with the `clone` label to a separate repository ([link](https://github.com/pimcore/pimcore/pull/16108/files?diff=unified&w=0#diff-a988c2a009da90c73aa7f74a09cf52baf9ccdc7869bce4b27f98feb27c3e5d29L1-R28))

### CODE REVIEW

**Review by herbertroth — APPROVED** [review]

File: .github/workflows/clone-pr.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - name: Check for AutomatedReleasePR label
+      id: label-check
+      run: |
+        HAS_LABEL=$(echo '$LABELS' | jq 'any(.[]; .name == "AutomatedReleasePR")')
+        echo "::set-output name=has_label::$HAS_LABEL"
+
+  clone-pr:
+    needs: check-labels
+    uses: pimcore/workflows-collection-public/.github/workflows/reusable-pr-clone.yml@v1.0.0-rc.1
+    if: github.event.pull_request.merged == true && github.repository == 'pimcore/pimocre' && needs.check-labels.outputs.has_release_label == 'false'
```
Comment (dvesh3) [inline .github/workflows/clone-pr.yml]:
```suggestion
    if: github.event.pull_request.merged == true && github.repository == 'pimcore/pimcore' && needs.check-labels.outputs.has_release_label == 'false'
```

**Review by dvesh3 — COMMENTED** [review]
I would like to know what happens when we push direct changes to the branch. How do we sync those changes?

**Comment by bluvulture** [comment]
> I would like to know what happens when we push direct changes to the branch. How do we sync those changes?

Dear @dvesh3 as explained in our teams chat.

**Comment by herbertroth** [comment]
@bluvulture We will need a detailed documentation, how to handle each daily task, available in Confluence before we can rollout the ee repos and the sync. This must include all new rules and workflows and a detailed overview of the "magic" happens with automations.

**Review by brusch — APPROVED** [review]

**Review by cipribucur — APPROVED** [review]

**Pull request merged**